python/lsst/fgcmcal/utilities.py

```diff
@@ -367,7 +367,7 @@ def translateVisitCatalog(visitCat):
                                                  ('TELDEC', 'f8'),
                                                  ('TELROT', 'f8'),
                                                  ('PMB', 'f8'),
-                                                 ('FILTERNAME', 'a10')])
+                                                 ('FILTERNAME', 'a30')])
```

Comment:
DECam physical labels can be over 30 characters. I think some of the LSST LATISS filters are about that length too.